api/fuzzing/README.md

```diff
@@ -0,0 +1,63 @@
+# API Fuzzer
+
+This project uses [RESTler](https://github.com/microsoft/restler-fuzzer) (from Microsoft) to fuzz the API.
+
+This is early days, there are still
```

Comment:
"there are still"?

Reply:
He's just got a way with words.

Reply:
Accidental inclusion, I'll remove this :P